index.html

```diff
@@ -10,29 +10,20 @@
 </head>
 <body>
     <section class="mobile">
-   <nav class="navbar">
-    
+   <nav class="navbar">  
       <div class="toolbar">
-        <div class="logo">My Logo</div>
+        <div class="logo"> <a href="#">My Logo</a>
+          </div>
         <img class="toggle" src="images/Union.png" alt="">
-      
-     <!-- DROPDOWN MENU 
-      <a class="humburger" href="#">&#9776;</a>
-          <ul class="dropdown">
-            <li><a href="#">portfolio </a></li>
-            <li><a href="#">About</a></li>
-            <li><a href="#">Contact</a></li>
-          </ul>
-      -->
       </div>
    
   </nav>
   <div class="headline">
     
-   <h2>
+   <h1>
     I’m Mariane Glad to see you!!
-   </h2>
-   <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+   </h1>
+   <h1>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</h1>
```

Comment:
Kindly, for accessibility your web page should only contain a single H1 tag, for this case use a P tag